core/src/main/java/hudson/WebAppMain.java

```diff
@@ -254,7 +254,10 @@ public Locale get() {
             // check that and report an error
             try {
                 File f = File.createTempFile("test", "test");
-                f.delete();
+                boolean result = f.delete();
+                if (!result) {
+                    LOGGER.log(WARNING, "Temp file test.test could not be deleted.");
```

Comment:
I believe that the WARNING level is on by default.

Wouldn't it be safer and less likely to spam the log if this were logged at a level that is off by default, like FINE?  Since the failure to delete has not been a reported problem in the past, it seems wise to not risk log spam on systems like Windows.
```suggestion
                    LOGGER.log(WARNING, "Temp file test.test could not be deleted.");
```

Reply:
Thanks Mark, I changed it as suggested 👍